src/components/backend-ai-resource-broker.ts

```diff
@@ -601,7 +603,57 @@ export default class BackendAiResourceBroker extends BackendAIPage {
       }
 
       this.total_slot = total_slot;
-      this.total_resource_group_slot = total_resource_group_slot;
+      if (!globalThis.backendaiclient._config.hideAgents) {
+        let initialResourceSlots = {
```

Comment:
This is small feedback. You can use `const` here.  😬 
Other option sis  don't assign the initial value to variable(`initialResourceSlots`) and just put that on reducer (649 line)

Reply:
Yes, It doesn't seem necessary to assign it as a local variable.